### PR TITLE
Moved new content from neetix to neetoUI

### LINKED ISSUE #1838: Improve Storybook documentation for neetoUI
- [x] [Using Storybook](https://neetix.neetokb.com/articles/using-storybook)
- [x] [Color](https://neetix.neetokb.com/articles/color)
- [x] [CSS variables](https://neetix.neetokb.com/articles/css-variables-support)
- [x] [Dark mode](https://neetix.neetokb.com/articles/dark-mode)
- [x] [Customise neetoUI](https://neetix.neetokb.com/articles/customise-neetoui)

@praveen-murali-ind _a

### PR DESCRIPTION
**Pull request by nitinprojects**
- Fixes #1838

**Description**

Content from the articles below has been copied to respective pages.
 
[Using Storybook](https://neetix.neetokb.com/articles/using-storybook)
 [Color](https://neetix.neetokb.com/articles/color)
 [CSS variables](https://neetix.neetokb.com/articles/css-variables-support)
 [Dark mode](https://neetix.neetokb.com/articles/dark-mode)
 [Customise neetoUI](https://neetix.neetokb.com/articles/customise-neetoui)

**Checklist**

- [ ] ~I have made corresponding changes to the documentation.~
- [ ] ~I have updated the types definition of modified exports.~
- [ ] ~I have verified the functionality in some of the neeto web-apps.~
- [ ] ~I have added tests that prove my fix is effective or that my feature works~
- [ ] ~I have added the necessary label (patch/minor/major - If package publish
      is required).~

**Reviewers**

### CODE REVIEW

**Comment by nitinprojects** [comment]
@praveen-murali-ind _a Please review.

File: stories/Customize/CSSVariables.stories.mdx [hunk 1]
```diff
@@ -172,7 +185,9 @@ a {
   </div>
 </Story>
 
-#### Dark theme [Experimental]
+<br/>
+
+### Dark theme
```
Comment (praveen-murali-ind) [inline stories/Customize/CSSVariables.stories.mdx:190]: @nitinprojects _a Replace "Dark theme" with "Dark mode"

File: stories/Customize/CSSVariables.stories.mdx [hunk 2]
```diff
@@ -172,7 +185,9 @@ a {
...
 
 <Story name="Dark theme">
```
Comment (praveen-murali-ind) [inline stories/Customize/CSSVariables.stories.mdx:192]: @nitinprojects _a Rename this story to "Dark mode".

**Review by praveen-murali-ind — COMMENTED** [review]
@nitinprojects _a A few minor changes are required. The rest looks good.

File: stories/Customize/CSSVariables.stories.mdx (hunk 2 above)
Comment (nitinprojects) [reply stories/Customize/CSSVariables.stories.mdx:192]: @praveen-murali-ind  can you update the name in CodeSandbox https://codesandbox.io/s/dark-theme-2dhsl0 and share the new code?

Comment (nitinprojects) [reply stories/Customize/CSSVariables.stories.mdx]: @praveen-murali-ind _a

**Review by praveen-murali-ind — APPROVED** [review]

**Pull request merged**